libcloud/common/base.py

```diff
@@ -587,6 +590,7 @@ def request(self, action, params=None, data=None, headers=None,
                     url=url,
                     body=data,
                     headers=headers,
+                    raw=raw,
```

Comment:
Why was this added?

Reply:
This is because [`prepared_request` uses raw instead of stream](https://github.com/apache/libcloud/blob/c2e71ecb4c2b97d858eebac8cae384438338f268/libcloud/http.py#L224-L238). This probably needs a refactoring to do since the stream parameter is unused, but I wanted to keep my change minimal and avoid disruption because changes here affect every driver.

Reply:
Ah, ok.